notary/content.md

```diff
@@ -0,0 +1,82 @@
+%%LOGO%%
+
+# How to use this repository
+
+The Notary respository contains two distinct applications, Notary Server, and Notary Signer. The images for these applications are tagged "server-\*" and "signer-\*" respectively. While the server can be configured to run entirely in memory, this configuration is not be appropriate for a production deployment so you should expect to run both a server _and_ and signer. 
+
+Ensure that the images you are running have similar version tags. That is, if you are running the server-0.2.0 tag, you should also be running the similar signer-0.2.0 tag. Running different versions of the server and signer will never be a supported configuration.
+
+# Notary Server
+
+The Notary server manages JSON formatted TUF (The Update Framework) metadata for Notary clients and the docker command line tool's Docker Content Trust features. It requires a companion Notary signer instance and a MySQL (or MariaDB) database.
+
+## How to use this image
+
+The following sample configuration is included in the image:
+
+	{
+	    "server": {
+	        "http_addr": ":4443",
+	        "tls_key_file": "/certs/notary-server.key",
+	        "tls_cert_file": "/certs/notary-server.crt"
+	    },
+	    "trust_service": {
+	      "type": "remote",
+	      "hostname": "notarysigner",
+	      "port": "7899",
+	      "tls_ca_file": "/certs/root-ca.crt",
+	      "key_algorithm": "ecdsa",
+	      "tls_client_cert": "/certs/notary-server.crt",
+	      "tls_client_key": "/certs/notary-server.key"
+	    },
+	    "logging": {
+	        "level": "info"
+	    },
+	    "storage": {
+	        "backend": "mysql",
+	        "db_url": "server@tcp(mysql:3306)/notaryserver?parseTime=True"
+	    }
+	}
+
+The components you *must* provide are the certificates and keys, and the links for the `notarysigner` and `mysql` hostnames. The `root-ca.crt` file enables the Notary server to identify valid signers, which it communicates with over mutual TLS using a GRPC interface. The `notary-server.crt` and`notary-server.key` are used to identify this service to both external clients, and signer instances.
+
+If you require a different configuration, you should wrap this image with your own Dockerfile.
+
+For more details on how to configure your Notary server, please read the[docs](https://github.com/docker/notary/blob/master/docs/notary-server-config.md).
+
+# Notary Signer
+
```

Comment:
Might want to mention about permissions on these cert files since the `notary` user in the container will need to be able to read them.